src/header.tsx

```diff
@@ -120,113 +135,101 @@ export const Header = () => {
-            {!accounts.length && (
-              <Button onClick={() => setIsConnectionDialiogOpen(true)}>
-                Connect
-              </Button>
-            )}
-            {!!accounts.length && (
-              <DropdownMenu>
-                <DropdownMenuTrigger asChild>
-                  <Button
-                    variant="outline"
-                    size="default"
-                    className="cursor-pointer overflow-hidden"
-                  >
-                    <Polkicon
-                      size={36}
-                      address={selectedAccount?.address || ''}
-                      className="mr-2"
-                      outerColor="transparent"
-                    />
-                    {selectedAccount?.name}
-                    <ChevronDown className="ml-2 h-4 w-4" />
-                  </Button>
-                </DropdownMenuTrigger>
-                <DropdownMenuContent>
-                  {accounts.map((account, index) => (
-                    <>
-                      <DropdownMenuItem
-                        className="cursor-pointer"
-                        key={account.address}
-                        onClick={() => selectAccount(account)}
-                      >
-                        <Polkicon
-                          size={28}
-                          address={account.address || ''}
-                          className="mr-2"
-                          outerColor="transparent"
-                        />
-                        {account.name}
-                      </DropdownMenuItem>
-                      {index !== accounts.length - 1 && (
-                        <DropdownMenuSeparator />
-                      )}
-                    </>
-                  ))}
-                  <DropdownMenuItem
+                <DropdownMenuSeparator />
+                <DropdownMenuItem>
+                  <div
                     className="cursor-pointer"
-                    key={'logout'}
                     onClick={() => {
-                      disconnectAll()
-                      selectAccount(undefined)
+                      setModalOpen(true)
                     }}
                   >
-                    Logout
-                  </DropdownMenuItem>
-                </DropdownMenuContent>
-              </DropdownMenu>
-            )}
-          </div>
+                    Show Wallets
+                  </div>
+                </DropdownMenuItem>
+              </DropdownMenuContent>
+            </DropdownMenu>
+          ) : (
+            <Button onClick={() => setModalOpen(true)}>Connect</Button>
+          )}
+        </div>
+        <div style={{ fontSize: '1rem' }}>
+          <ConnectModal
+            type="extensions"
+            config={connectConfig}
+            selected={selectedAccount}
+            setSelected={selectAccount}
+            getConnectedAccounts={(acc) => {
+              setConnectedAccounts(acc)
+            }}
```

Comment:
Same, I wouldn't be able to guess from the name what this does, or why `setConnectedAccounts` is the right func for this.
```suggestion
            getConnectedAccounts={setConnectedAccounts}
```